src/generic/LoadingPage.jsx

```diff
@@ -2,34 +2,38 @@ import React from 'react';
 import { Col } from '@edx/paragon';
 import PropTypes from 'prop-types';
 
-const LoadingPage = ({ loadingMessage }) => (
-  <Col
-    xs={12}
-    className="justify-content-center d-flex"
-    style={{
-      height: '50vh',
-    }}
-  >
-    <div className="spinner-border text-primary text-center align-self-center" role="status">
-      {loadingMessage && (
-      <span className="sr-only">
-        {loadingMessage}
-      </span>
-      )}
-    </div>
-  </Col>
-);
+function LoadingPage({ loadingMessage }) {
```

Comment:
It seems all of the `const` components have been replaced with `function`

Is there something in the linter that's requiring that?

Reply:
Yes, this is due to satisfaction of an automatically fixable `react/function-component-definition` rule. You can find more details on the rule [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md).